src/core/RenderTarget.js

```diff
@@ -34,6 +34,7 @@ class RenderTarget extends EventDispatcher {
 			minFilter: LinearFilter,
 			depthBuffer: true,
 			stencilBuffer: false,
+			ignoreDepthValues: false,
```

Comment:
Is this an undocumented property that users do not set? Perhaps it should it be "private" in some form.

> `ignoreDepthValues: false`

Also, I am not in fan of double-negatives. Maybe `_resolveDepthBuffer`, or something similar.

Reply:
I'm going to add a documentation. Similar to `resolveStencilBuffer` this property is for advanced use cases.

I've picked `ignoreDepthValues` as a name so it matches the same property of the WebXR spec. 

https://developer.mozilla.org/en-US/docs/Web/API/XRWebGLLayer/ignoreDepthValues

TBH, I think the name is fine. Besides, the property does not just control the MSAA resolve. The ignore also means the respective buffers are invalidated (if the browser supports framebuffer invalidation).

Reply:
We also need to evaluate this property in `WebGPURenderer` at some point. Not having a property in `RenderTarget` means redundant code snippets like the following will be spread in both renderers multiple times.

```js
const renderTargetProperties = properties.get( renderTarget );
const ignoreDepthValues = ( renderTargetProperties.__ignoreDepthValues !== undefined ) ? renderTargetProperties.__ignoreDepthValues : false;
```

Reply:
> TBH, I think the name is fine. Besides, the property does not just control the MSAA resolve. The ignore also means the respective depth buffers are invalidated (if the browser supports framebuffer invalidation).

On the other hand, the framebuffer invalidation is some sort of clean up that is only happens in the MSAA codepath now. Um, I'll add a commit with `resolveDepthBuffer` for comparison.

Reply:
@WestLangley Looking better now?

Reply:
@Mugen87 I like that better, thanks.

Let's leave this review section open for now, so others can see your explanations. 👍